sql/step_94_missing_user_related_fk.sql

```diff
@@ -0,0 +1,550 @@
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PATIENT 
+ADD INDEX FK_PATIENT_CREATED_BY_IDX (PAT_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PATIENT 
+ADD CONSTRAINT FK_PATIENT_CREATED_BY
+  FOREIGN KEY (PAT_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PATIENTEXAMINATION 
+ADD INDEX FK_PATIENTEXAMINATION_CREATED_BY_IDX (PEX_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PATIENTEXAMINATION 
+ADD CONSTRAINT FK_PATIENTEXAMINATION_CREATED_BY
+  FOREIGN KEY (PEX_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PATIENTHISTORY 
+ADD INDEX FK_PATIENTHISTORY_CREATED_BY_IDX (PAH_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PATIENTHISTORY 
+ADD CONSTRAINT FK_PATIENTHISTORY_CREATED_BY
+  FOREIGN KEY (PAH_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PATIENTVACCINE 
+ADD INDEX FK_PATIENTVACCINE_CREATED_BY_IDX (PAV_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PATIENTVACCINE 
+ADD CONSTRAINT FK_PATIENTVACCINE_CREATED_BY
+  FOREIGN KEY (PAV_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PERMISSIONS 
+ADD INDEX FK_PERMISSIONS_CREATED_BY_IDX (P_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PERMISSIONS 
+ADD CONSTRAINT FK_PERMISSIONS_CREATED_BY
+  FOREIGN KEY (P_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PREGNANTTREATMENTTYPE 
+ADD INDEX FK_PREGNANTTREATMENTTYPE_CREATED_BY_IDX (PTT_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PREGNANTTREATMENTTYPE 
+ADD CONSTRAINT FK_PREGNANTTREATMENTTYPE_CREATED_BY
+  FOREIGN KEY (PTT_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PRICELISTS 
+ADD INDEX FK_PRICELISTS_CREATED_BY_IDX (LST_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PRICELISTS 
+ADD CONSTRAINT FK_PRICELISTS_CREATED_BY
+  FOREIGN KEY (LST_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PRICES 
+ADD INDEX FK_PRICES_CREATED_BY_IDX (PRC_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PRICES 
+ADD CONSTRAINT FK_PRICES_CREATED_BY
+  FOREIGN KEY (PRC_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_PRICESOTHERS 
+ADD INDEX FK_PRICESOTHERS_CREATED_BY_IDX (OTH_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_PRICESOTHERS 
+ADD CONSTRAINT FK_PRICESOTHERS_CREATED_BY
+  FOREIGN KEY (OTH_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+ALTER TABLE OH_SESSION_AUDIT 
+ADD INDEX FK_SESSION_AUDIT_CREATED_BY_IDX (SEA_CREATED_BY ASC) VISIBLE;
+
+ALTER TABLE OH_SESSION_AUDIT 
+ADD CONSTRAINT FK_SESSION_AUDIT_CREATED_BY
+  FOREIGN KEY (SEA_CREATED_BY)
+  REFERENCES OH_USER (US_ID_A)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
```

Comment:
This should be removed because override the previous one

Reply:
sorry, I missread the code... this one can stay but should be CASCADE as well

Reply:
You're right. It would not make sense to leave this one as no_action if we have cascade in the other.

Reply:
Solved by https://github.com/informatici/openhospital-core/pull/998/commits/78e084d4ec8e39ff6dbc85f99b83a49f96cc7cae